### PR TITLE
Fix deploy and more logs tweaks

### PR DESCRIPTION
**Pull request by bloodearnest**
- **Fix image name that had changed with move to docker compose**
- **Include the host name in the access logs**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -15,8 +15,12 @@ http {
     include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
 
-    log_format  main  'ts=$time_iso8601 status=$status req="$request" proxy=$proxy_host '
-                      'ms=$request_time bytes=$bytes_sent '
+    # custom log format
+    #  - more readable format 
+    #  - includes proxy host
+    #  - includes virtual host, because we only have one log stream, and multiple hosts
+    log_format  main  'ts=$time_iso8601 status=$status method=$request_method uri=$host/$uri?$args proxy=$proxy_host '
```
Comment (evansd) [inline nginx.conf:22]: I think you can use [`$is_args`](https://nginx.org/en/docs/http/ngx_http_core_module.html#var_is_args) if you only want to include a `?` if there are query parameters, but that's probably a bit "extra".

Alternatively, would logging [`$request_uri`](https://nginx.org/en/docs/http/ngx_http_core_module.html#var_request_uri) work here to get the whole thing in one go?

**Review by evansd — APPROVED** [review]

File: nginx.conf (hunk 1 above)
Comment (bloodearnest) [reply nginx.conf:22]: request_uri works, thanks

Comment (bloodearnest) [reply nginx.conf:22]: Although, it appends ? to everything, so maybe is_args is better?

Comment (bloodearnest) [reply nginx.conf:22]: oh, no it only does that so one of the tests logs.. Never mind.

**Pull request merged**